ReactWindows/ReactNative/Views/TextInput/ReactTextInputManager.cs

```diff
@@ -25,6 +26,15 @@ class ReactTextInputManager : BaseViewManager<ReactTextBox, ReactTextInputShadow
 
         private bool _onSelectionChange;
 
+        private static readonly Color DefaultTextBoxBorder =
+            new Color
+            {
+                A = byte.Parse("FF", NumberStyles.AllowHexSpecifier),
+                R = byte.Parse("7A", NumberStyles.AllowHexSpecifier),
+                G = byte.Parse("7A", NumberStyles.AllowHexSpecifier),
+                B = byte.Parse("7A", NumberStyles.AllowHexSpecifier)
+            };
```

Comment:
I get what you're doing here, but what about just Color.FromArgb(255, 122, 122, 122)?